rs/consensus/src/consensus/priority.rs

```diff
@@ -44,39 +44,39 @@ fn compute_priority(
     notarized_height: Height,
```

Comment:
Can we rename this function to `computer_bouncer_value`? (or something like that)